### ISSUE TITLE
Does bar still lack XFT support?

### ISSUE THREAD

**Issue by vendion** [LemonBoy/bar#188]
I know there has been a lot of back and forth about this issue, but where things ended up are fairly unclear.  I'm trying to use bar with two TTF fonts "Roboto Slab" and "FontAwesome", both are installed on my system:

```
[vendion@Fail4][~]% fc-list | grep 'Roboto Slab'
/usr/share/fonts/TTF/RobotoSlab-Light.ttf: Roboto Slab:style=Light
/usr/share/fonts/TTF/RobotoSlab-Regular.ttf: Roboto Slab:style=Regular
/usr/share/fonts/TTF/RobotoSlab-Thin.ttf: Roboto Slab:style=Thin
/usr/share/fonts/TTF/RobotoSlab-Bold.ttf: Roboto Slab:style=Bold
[vendion@Fail4][~]% fc-list | grep 'FontAwesome'
/usr/share/fonts/TTF/fontawesome-webfont.ttf: FontAwesome:style=Regular
[vendion@Fail4][~]% 
```
I have even tried different syntaxes for telling bar to use them `-f "Roboto Slab-10" -f "FontAwesome-10"`, `-f "Roboto Slab:size=10" -f "FontAwesome:size=10"`, and `-f "xft:Roboto Slab:size=10" -f "FontAwesome:size=10"`.  I've even tried this syntax `-*-Roboto Slab-medium-o-*-*-14-*-*-*-*-*-*-*` to no avail.  This leads me to think XFT is still unsupported in this version, is this correct?

**Comment by onodera-punpun** [comment]
yes, use https://github.com/geommer/yabar or https://github.com/krypt-n/bar instead.

**Comment by rmhsilva** [comment]
Hi, sorry to bring up this old issue, but as it's still open... Are there any plans to merge the krypt-n XFT fork back in here?
Cheers

**Comment by preisi** [comment]
I would also like to know, if there are any plans to finally include xft-support. The cairo-love branch is quite old already :/

**Comment by preisi** [comment]
any updates?

**Comment by alecive** [comment]
Ping!

**Comment by quigley-c** [comment]
Is there a particular reason xft hasn't been considered? Maybe it's unnecessary considering the original goals of bar?

**Comment by ChristopherTheBoss** [comment]
Bump

**Comment by domsson** [comment]
Not the author, but as far as I understand, XFT support does add quite a bit of code and code complexity. However, not everyone needs the feature. Lemonbar currently sets itself apart from other projects, like Polybar, mainly due to its minimalistic approach and light footprint. Therefore, I believe that LemonBoy doesn't want to merge this. But again, this is just what I believe to have read elsewhere, I'm not 100%. I do feel this way myself, however.

**Comment by silentz** [comment]
Created an up-to-date (Oct 2024) fork of lemonbar with applied Xft support:
https://github.com/silentz/lemonbar-xft

The fork was made because other forks have not been updated for a long time, and they lack multimonitor support

**Comment by lexiwitch** [comment]
bumping this 9 year old issue because it is still relevant
